app/components/Message.tsx

```
@@ -0,0 +1,204 @@
+                      initial={{ height: 0, opacity: 0 }}
+                      animate={{ height: "auto", opacity: 1 }}
+                      exit={{ height: 0, opacity: 0 }}
+                      transition={{ duration: 0.1 }}
+                    >
+                      <li className={styles.messageMenuItem}>
+                        <button
+                          onClick={() => {
+                            setIsMenuOpen(false);
+                            handleDeleteMessage();
+                          }}
+                        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)